code/game/objects/items/storage/storage.dm

```diff
@@ -721,24 +720,25 @@ W is always an item. stop_warning prevents messaging. user may be null.**/
 			to_chat(user, SPAN_WARNING("[ammo_dumping] is empty."))
 	return TRUE
 
-/obj/item/storage/proc/dump_into(obj/item/storage/M, mob/user)
+/obj/item/storage/proc/dump_into(obj/item/storage/Main, mob/user)
```

Comment:
Two things:
- `Main` makes no sense to be the variable name; this is kinda more confusing than just a random letter like it used to be.
- We use snake_case; not camelCase or PascalCase. Your variable here is PascalCase. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case

So something more like `obj/item/storage/origin_storage` would be more appropriate.

Reply:
My bad put it there as a placeholder i replace it with origin_storage like you suggest.